packages/components/src/public/files/FileAttachmentForm.tsx

```diff
@@ -379,7 +377,7 @@ export class FileAttachmentForm extends React.Component<FileAttachmentFormProps,
         );
     }
 
-    render(): ReactNode {
+    render(): JSX.Element {
```

Comment:
was there a reason for this change? I would expect the return type here to remain as `ReactNode`.

Reply:
Reason for this change was due to this error:

ERROR in C:\labkey\trunkGit\server\modules\wnprc-modules\WNPRC_Purchasing\src\client\components\DocumentAttachmentPanel.tsx
./src/client/components/DocumentAttachmentPanel.tsx
[tsl] ERROR in C:\labkey\trunkGit\server\modules\wnprc-modules\WNPRC_Purchasing\src\client\components\DocumentAttachmentPanel.tsx(39,22)
      TS2786: 'FileAttachmentForm' cannot be used as a JSX component.
  Its instance type 'FileAttachmentForm' is not a valid JSX element.
    The types returned by 'render()' are incompatible between these types.
      Type 'ReactNode' is not assignable to type 'false | Element'.
        Type 'string' is not assignable to type 'false | Element'.

Reply:
Hmm, interesting. I'd like to look at this with you if you have a chance @labkey-bpatel.

Reply:
Alright, so I investigated further and the remedy is to rehydrate that WNPRC module's `package-lock.json`. Doing this resolved this issue. I bumped `@labkey/components` and then rehydrated. These changes were pushed to your WNPRC branch. https://github.com/LabKey/wnprc-modules/pull/62/commits/7ba41f62bc9.

Reply:
This is now reverted back to using ReactNode

Reply:
Thank you, Nick for troubleshooting & fixing